package.json

```diff
@@ -49,7 +49,6 @@
     "semver": "7.3.5",
     "shx": "0.3.3",
     "ts-node": "9.1.1",
-    "ts-transformer-inline-file": "0.1.1",
```

Comment:
You should commit the changed package-lock.json file as well.

Reply:
For some reason there weren't any available to commit. `npm run reinstall` seemed to do the trick.